skiko/src/commonMain/kotlin/org/jetbrains/skia/Expects.kt

```diff
@@ -6,8 +6,6 @@ internal expect fun String.intCodePoints(): IntArray
 
 internal expect fun defaultLanguageTag(): String
 
-expect abstract class OutputStream
```

Comment:
Note: Public API breaking change. 

`class OutputWStream` was moved to jvmMain only. we have its usage only in jvm tests - SVGCanvasTest.
It's not obvious how to map the bindings to make them work for web targets (native too? we probably need our own abstraction in C bindings for OutputStream which would mimic the  OutputStream in java). 

Decided to keep it simple for now and moved it to jvmMain.